src/elements/Button/Button.styl

```diff
@@ -1,8 +1,19 @@
 @import "~@/assets/tokens/tokens.styl"
-
 @import "~@/styles/colors.styl"
 @import "~@/styles/font-reset.styl"
 @import "~@/styles/sizes.styl"
+@import "~@/styles/control.styl"
+@import "~@/patterns/LicenseBadge/LicenseBadge.styl"
+
+
+ // Logo Assign CC BY-SA
+ 
+.vocab.license-badge
```

Comment:
Styling for license badges should stay in `LicenseBadge.styl`.

Reply:
No problem. This addition can be removed.

Reply:
```suggestion

```